src/controllers/language-listener/language-listener.js

```diff
@@ -0,0 +1,22 @@
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+export class LanguageListenerController {
+
+	constructor(host) {
+		(this.host = host).addController(this);
+		this._localeSettings = getDocumentLocaleSettings();
+		this._handleLangUpdate = () => {
+			this.language = this._localeSettings.language;
+			this.host.requestUpdate();
+		};
+		this.language = this._localeSettings.language;
+	}
+
+	hostConnected() {
+		this._localeSettings.addChangeListener(this._handleLangUpdate);
+	}
+
+	hostDisconnected() {
+		this._localeSettings.removeChangeListener(this._handleLangUpdate);
+	}
+}
```

Comment:
Not a big deal but seems clearer to me that each instance doesn't actually have its own locale settings this way:
```suggestion
const localeSettings = getDocumentLocaleSettings();

export class LanguageListenerController {

	constructor(host) {
		(this.host = host).addController(this);
		this._handleLangUpdate = () => {
			this.language = localeSettings.language;
			this.host.requestUpdate();
		};
		this.language = localeSettings.language;
	}

	hostConnected() {
		localeSettings.addChangeListener(this._handleLangUpdate);
	}

	hostDisconnected() {
		localeSettings.removeChangeListener(this._handleLangUpdate);
	}
}
```

Reply:
Yikes, that's not a helpful diff at all. Just broke `this._localeSettings` outside the class to `const localeSettings = getDocumentLocaleSettings();` and referenced that within the class.